<launch>
	<arg name="marker_size" default="6" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />

    <arg name="cam_image_topic" default="panda_autograsp/kinect2/hd/points" />
	<arg name="cam_info_topic" default="panda_autograsp/kinect2/hd/camera_info" />
	<arg name="output_frame" default="kinect2_rgb_optical_frame" />
	<arg name="bundle_files" default="$(find panda_autograsp)/bundles/MarkerData_8_9_10_2.xml" />
    <arg name="med_filt_size" default="10" />

	<node name="ar_track_alvar" pkg="ar_track_alvar" type="findMarkerBundles" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame) $(arg med_filt_size) $(arg bundle_files)" />
</launch>
